asmcomp/amd64/emit.mlp

```diff
@@ -464,16 +459,15 @@ let add_float_constant cst =
     lbl
 
 let emit_float_constant f lbl =
-  _label (emit_label lbl);
-  D.qword (Const f)
-
-let emit_global_label s =
-  let lbl = Compilenv.make_symbol (Some s) in
-  add_def_symbol lbl;
-  let lbl = emit_symbol lbl in
-  D.global lbl;
-  _label lbl
+  D.define_label lbl ~typ:NONE;
```

Comment:
I don't know if this matters but all instances of `_label` should be translated to `~typ:QWORD`, no?